openstack/dns/v2/quotas/requests.go

```diff
@@ -0,0 +1,63 @@
+package quotas
+
+import (
+	"context"
+
+	"github.com/gophercloud/gophercloud/v2"
+)
+
+// ListOptsBuilder allows extensions to add parameters to the List request.
+type ListOptsBuilder interface {
+	ToQuotasListQuery() (string, error)
+}
+
+type ListOpts struct {
+}
+
+// ToZoneListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToQuotasListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+// Get returns information about the quota, given its ID.
```

Comment:
Perhaps we can clarify a bit
```suggestion
// Get returns information about the quota for a given project ID.
```